includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -147,7 +147,7 @@ public function get_total_label() {
 	 * @param bool|null $is_deposit      Whether this is a deposit.
 	 * @param int       $deposit_plan_id Deposit plan ID.
 	 *
-	 * @return integer Total price.
+	 * @return int|float Total price.
 	 */
 	public function get_product_price( $product, $is_deposit = null, $deposit_plan_id = 0 ) {
```

Comment:
Hum, what if we keep it more consistent/strict, always returning the same type? Either we cast anything to float or int when returning

Reply:
Addressed in [3162320](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3617/commits/3162320d37fc5709cfe6eb38f0c567d4fa9457f6)